### DISCUSSION TITLE
Help needed with dagster-webserver, possibly related to ipv4 vs ipv6

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17873] [Q&A]
When running the dagster-webserver on my kubernetes cluster, I am running into issues where the `dagster`  container of the dagster-webserver pod is failing (image: `<http://docker.io/dagster/dagster-celery-k8s:1.5.6|docker.io/dagster/dagster-celery-k8s:1.5.6>`). Specifically, the error is here:
```
2023-11-07T22:34:59.696378Z	error	Request to probe app failed: Get "http://[2602:fb33:0:12:3750::b]:80/server_info": dial tcp [::6]:0-&gt;[2602:fb33:0:12:3750::b]:80: connect: connection refused, original URL path = /app-health/dagster/readyz
```

I think that the error is stemming from ipv4 vs ipv6 issues since the dagster webserver is running here in the deployment:

```
    - command:
        - /bin/bash
        - -c
        - ' dagster-webserver -h 0.0.0.0 -p 80 -w /dagster-workspace/workspace.yaml'
```

since this runs on ipv4 and not ipv6. However, I tried modifying this to be
``` 
    - command:
        - /bin/bash
        - -c
        - ' dagster-webserver -h [::] -p 80 -w /dagster-workspace/workspace.yaml'
```

manually within the deployment yaml and this didn't seem to fix the underlying issue. Any ideas here? Have others ran into this in the past? I saw this github issue but it looks like no <https://github.com/dagster-io/dagster/issues/15525|actions> have been taken here

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C014N0PK37E/p1699400722447039?thread_ts=1699400722.447039&cid=C014N0PK37E).*

**Comment by johannkm** [comment]
https://github.com/dagster-io/dagster/issues/15525 is the correct link for the issue

**Reply by ian-scale** [reply]
@johannkm is there any plan for this to work in the future? Or any current workarounds that I could try to support dagit on a ipv6 k8s cluster?

**Comment by ian-scale** [comment]
for anyone else who is having trouble with this, a current workaround is to set `logLevel: "warning -h '::'"` on the webserver.
